CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+- Add new `no-trailing-spaces` rule.
```

Comment:
I like to give an example of good and bad, but it seems hard for this one (since whitespace would not be displayed anyways...

Reply:
added examples best I could